src/components/todoList/TodoList.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Todo } from '../todo';
+
+export const TodoList = ({ preparedTodos }) => (
+  preparedTodos.map(ele => (
```

Comment:
ele - некорректное название
```suggestion
  preparedTodos.map(todo => (
```

Reply:
дякую, виправив)